### ISSUE TITLE
[Calculation] Crash when opening additional results 

### ISSUE THREAD

**Issue by lolocomotive** [UpsilonNumworks/Upsilon#253]
The (linux) simulator crashes when you open the additional results panel when making a trigonometry calculation.

**To Reproduce**
- Go to calculations
- type cos(0)
- open additional results
- Crash

Works fine on the calculator tho...

Crashes since c826e556a12deb89e33a121102ae55cd40a0b907

ion/src/shared/storage.cpp:64
```cpp
const Record metadataRecord = recordBaseNamedWithExtension(record.fullName(), "sys");
``` 
which calls ion/src/shared/internal_storage.cpp:603
```cpp
size_t nameLength = baseNameLength < 0 ? strlen(baseName) : baseNameLength;
```
which makes a segfault

**Comment by Yaya-Cout** [comment]
`baseName` is a pointer to `0x0` if I remember correctly. It's the `strlen` a call that crash on this line.

**Comment by Lauryy06** [comment]
It's supposed to be solved by 125e1a8a82dcd05a9ef8b93117a37c68cf044e63.
